cli/printer/text_test.go

```diff
@@ -356,45 +345,40 @@ func ExampleTextPrintTasks() {
 			TaskName:        "tsk1",
 			EnvironmentID:   "eid1",
 			EnvironmentName: "ename1",
-			RunningCount:    1,
-			DesiredCount:    1,
 			DeployName:      "d1",
 			DeployVersion:   "1",
+			Status:          "RUNNING",
```

Comment:
Is there any other status apart from RUNNING?

Reply:
http://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_life_cycle.html